source/api/http/namespace.rst

```diff
@@ -129,3 +129,26 @@ HTTP requests and responses have headers, according to the HTTP specification. H
       ["Content-Length"] = '32',
       ["Authorization"] = 'Bearer IssuedBearerToken'
    }
+
+Usage Example
+*************
+
+To send a HTTP GET request and read the response
```

Comment:
Change to `To send an HTTP GET request...`. Also full stop missing at end of line.